justfile

```diff
@@ -66,7 +66,7 @@ fmt:
 
   if command -v cargo-fmt >/dev/null; then
     echo '==> Running rustfmt'
-    cargo +nightly fmt --all -- --check
+    cargo +nightly fmt --all
```

Comment:
Why the change? This was used for validation, along with clippy to make everything is all right before pushing. Formatting is usually done on the fly by the editor.

Reply:
I see. It hasn't been a problem for me, ever, in VSCode. Maybe you forgot to enable the `+nightly` feature in your VSCode addons?

Reply:
I can check this, but either way, wouldn't it be better to auto-format instead of just checking? What if my VS Code is out-of-sync with my CI/just tooling?

Reply:
Sure, as long as we still check formatting on the CI.

Reply:
Absolutely! No doubt about that :)